app/Http/Controllers/Api/V1/Integration/BrightcoveAPISettingsController.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * @Author      Asim Sarwar
+ * Description  Handle Brightcove API Setting
+ * ClassName    BrightcoveAPISettingsController
+ */
+
+namespace App\Http\Controllers\Api\V1\Integration;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\App;
+use App\Models\Organization;
+use App\Models\Integration\BrightcoveAPISetting;
+use App\Http\Requests\V1\Integration\StoreBrightcoveAPISetting;
+use App\Http\Requests\V1\Integration\UpdateBrightcoveAPISetting;
+use App\Repositories\Integration\BrightcoveAPISettingRepository;
+use App\Http\Resources\V1\Integration\BrightcoveAPISettingCollection;
+use App\Http\Resources\V1\Integration\BrightcoveAPISettingResource;
+use App\Jobs\CloneBrightcoveAPISetting;
+
+class BrightcoveAPISettingsController extends Controller
+{
+    private $bcAPISettingRepository;
+    /**
+     * BrightcoveAPIClientController constructor.
+     * @param BrightcoveAPISettingRepository $brightcoveAPISettingRepository
+     */
+    public function __construct(BrightcoveAPISettingRepository $brightcoveAPISettingRepository)
+    {
+        $this->bcAPISettingRepository = $brightcoveAPISettingRepository;
+    }
+
+    /**
+     * Get All Brightcove API Settings for listing.
+     * Returns the paginated response with pagination links (DataTables are fully supported - All Params).
+     * @queryParam Organization $suborganization
```

Comment:
change @queryParam->@param for organization
other two @queryParam change to @bodyParam